docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,266 @@
+ddev config set extras $HOME/dd/integrations-extras
+ddev config set repo extras
+{{< /code-block >}}
+
+If you used a directory other than `$HOME/dd` to clone the integrations-extras directory, use the following command to set your working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set extras <PATH/TO/INTEGRATIONS_EXTRAS>
+ddev config set repo extras
+{{< /code-block >}}
+
+## Populate the integration tile scaffolding
+
+For Datadog API integrations and custom dashboard widgets that will be available out-of-the-box on the [Integrations page][102], use the Datadog Development Toolkit to create scaffolding for an informational tile-only listing.
+
+1. Make sure you're inside the `integrations-extras` directory:
+   {{< code-block lang="shell" >}}cd $HOME/dd/integrations-extras{{< /code-block >}}
+2. Run the `ddev` command with the `-t tile` option
+   {{< code-block lang="shell" >}}ddev create -t tile "<Offering Name>"{{< /code-block >}}
+
+[101]: https://docs.datadoghq.com/developers/integrations/python
+[102]: https://github.com/Datadog/integrations-extras
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Complete the necessary integration asset files
+
+Make sure that the following required assets for your integration are complete:
+
+{{% integration-assets %}}
+
+### README
+
+Once you have created a `README.md` file, add the following sections as H2s (`##`) and fill out the content accordingly:
+
+| Header Name | Header |
+|-------------|--------|
+| Overview | Write a description under an `## Overview` header that describes the value and benefits your offering provides to users, for example, out-of-the-box dashboards, replays of user sessions, logs, alerts, and more). <br><br>This information is displayed in the **Overview** tab on the tile. |
+| Setup | Include all the steps required to set up your offering that includes information divided into H3 headings (`###`). Standard topics include:<br><br>- Installing the integration using the in-app integration tile. <br>- Configuring the integration with the appropriate roles and permissions in your Datadog organization.<br>- Accessing out-of-the-box Datadog features that users who purchased and installed the integration can access (such as metrics, events, monitors, logs, dashboards, and more).|
+| Uninstallation | Include all the steps for uninstalling your offering. This information is displayed in the **Configure** tab on the tile.|
+| Data Collected  | Specify the types of data collected by your integration (if applicable), including events, service checks, logs, etc. Metrics added to the `metadata.csv` file will automatically appear in this tab.  <br><br> If your offering does not provide any of this data, you do not need to add a Data Collected section. |
+| Support | Provide contact information that includes an email to your Support team, a phone number to your company, a link to your company's documentation or blog post, and more help information in a bulleted list format. |
```

Comment:
Do we actually want to recommend they include a phone number?

Reply:
removed